great_expectations/data_context/types.py

```diff
@@ -0,0 +1,56 @@
+from collections import namedtuple
+from six import string_types
+
+from ..types import LooselyTypedDotDict
+
+NormalizedDataAssetName = namedtuple("NormalizedDataAssetName", [
+    "datasource",
+    "generator",
+    "generator_asset"
+])
+
+#Note: As of 2018/8/14, this class is VERY MUCH a work in progress
```

Comment:
Strike comment, or is this still too WIP?

Reply:
Def. want to strike before merging to develop. But it still feels pretty wippy to me.

The name **definitely** has to go.
-> `ValidationResultSuiteIdentifier`

Reply:
The flag here is our cost is multiple reviews of the same code.